### DISCUSSION TITLE
setMergeCompiledIncludes / merge_compiled_includes in Smarty 5

### DISCUSSION THREAD

**Discussion by Kalerindel** [smarty-php/smarty#990] [General]
Hi,

According to ([TODO](https://github.com/smarty-php/smarty/blob/34adf4e54cae4a352d3b239f18aacc8d04861921/TODO.txt#L12)), merging compiled templates with setMergeCompiledIncludes / merge_compiled_includes is currently not available in Smarty 5.

Can see the original note is from 8 months ago, so just wanted to ask if there have been any plans as of late to add it.

Was hoping to upgrade our codebase to PHP 8.3 and also move to v5, but the lack of it is currently a showstopper.

Cheers

**Comment by wisskid** [comment]
I didn't look into it yet, and I doubt anyone else has. Is there still a noticeable performance gain with the newer PHP versions with optimized opcode caches?
